docker-hub/roles-and-permissions.md

```diff
@@ -0,0 +1,76 @@
+
+The following roles are available to assign:
+
+- **Member** - Non-administrative role. Members can view other members that are in the same organization.
+- **Editor** - Partial administrative access to the organization. Editors can create, edit, and delete repositories. They can also edit an existing team's access permissions.
+- **Organization owner** - Full organization administrative access. Organization owners can manage organization repositories, teams, members, settings, and billing.
+- **Company owner** - In addition to the permissions of an organization owner, company owners can configure settings for their associated organizations.
+
+Owners can manage roles for members of an organization on [Docker Hub](/docker-hub/members/#update-a-member-role), as well members of an [organization](/admin/organization/members/#update-a-member-role) or a [company](/admin/company/users/#update-a-member-role) on [Docker Admin](/admin/).
+
+## Permissions
+
+The following sections describe the permissions for each role.
+
+### Content and registry permissions
+
+The following outlines content and registry permissions for member, editor, and organization owner roles. Company owners have the same access as organization owners for all associated organizations. See [Company overview](/admin/company/).
+
+| Permission | Member | Editor | Organization owner |
+|:----------------------- |:------ |:-------|:------------------ |
+| Explore images and extensions | ✅ | ✅ | ✅ |
+| Star, favorite, vote, and comment on content | ✅ | ✅ | ✅ |
+| Pull images | ✅ | ✅ | ✅ |
+| Create and publish an extension | ✅ | ✅ | ✅ |
+| Become a Verified, Official, or Open Source publisher | ❌ | ❌ | ✅ |
+| Observe content engagement as a publisher | ❌ | ❌ | ✅ |
+| Create public and private repositories | ❌ | ✅ | ✅ |
+| Edit and delete repositories | ❌ | ✅ | ✅ |
+| Manage tags | ❌ | ✅ | ✅ |
+| View repository activity | ❌ | ❌ | ✅ |
+| Set up Automated builds | ❌ | ❌ | ✅ |
+| Edit build settings | ❌ | ❌ | ✅ |
+| Set up vulnerability analysis with Docker Scout | ❌ | ✅ | ✅ |
+| View teams | ❌ | ✅ | ✅ |
+| Assign team permissions to repositories | ❌ | ✅ | ✅ |
+
+When you add members to a team, you can manage their repository permissions. For team repository permissions, see [Create and manage a team permissions reference](/docker-hub/manage-a-team/#permissions-reference).
+
+### Organization management permissions
+
+The following outlines organization management permissions for member, editor, organization owner, and company owner roles.
+
+| Permission | Member | Editor | Organization owner | Company owner |
+|:----------------------- |:------ |:-------|:------------------ |:----------- |
+| Create teams | ❌ | ❌ | ✅ | ✅ |
+| Manage, including delete, teams | ❌ | ❌ | ✅ | ✅ |
```

Comment:
This is very subjective, but it took me a few seconds to understand the meaning.
I don't know if `Manage teams, including delete` would be better because I don't have fresh eyes anymore.
Or the structure like the line below. `Manage teams (including delete)`

Reply:
Updating! I like the last option.